util/chunk/chunk_test.go

```diff
@@ -394,6 +394,43 @@ func (s *testChunkSuite) TestGetDecimalDatum(c *check.C) {
 	c.Assert(decDatum.Frac(), check.Equals, decFromChk.Frac())
 }
 
+func (s *testChunkSuite) TestChunkMemoryUsage(c *check.C) {
```

Comment:
Could you add case for date/time/datatime/blob ?

Reply:
blob has no difference with varchar
I add cases for duration and time